dcc-portal-ui/app/scripts/downloader/js/request.js

```diff
@@ -136,8 +136,10 @@
 
         $scope.params.processing = false;
       },function (error) {
-        $scope.params.processing = false;
-        $scope.params.downloadEnabled = false;
+        if(error.status === 503){
+          $scope.params.processing = false;
+          $scope.params.downloadEnabled = false;
+        }
```

Comment:
Discussion here:

http://stackoverflow.com/questions/28010548/restangular-how-to-override-error-interceptors